utils/errors.go

```diff
@@ -33,10 +33,10 @@ func (e ProcessingError) Cause() error {
 	return e.OriginalError
 }
 
-// ProcessingErrorFromErrorWithMessage converts an error into a ProcessingError
 func ProcessingErrorFromErrorWithMessage(
 	err error,
 	message string,
+	kafkaMessage kafka.Message,
```

Comment:
I think this param is unused or not being assigned in the processing error

Reply:
Great catch. I defaulted this to an empty message in one of the other 2 PRs because this message wasn't available until this change. I mapped it correctly and it is now used.